tests/test_mu.py

```diff
@@ -77,6 +78,7 @@ def test_publishes_a_lambda(self):
         result = mgr.publish(func)
         self.assertEqual(result["CodeSize"], 169)
 
+    @pytest.mark.skipif(sys.platform.startswith("win"), reason="Requires changes for Windows")
```

Comment:
could you add some comments on the issues for windows with this test or confirm in practice it is idempotent wrt to provisioning (--verbose to see that it doesn't re-provision), our zipfile assembly does try to remove most common causes for variance (empty timestamps, etc), so we get a hash of contents. i wonder if its an issue with the test record per se.

Reply:
It is a checksum failure vs recorded checksum -> attempted update -> test fails due to missing update recording

Let me dig in some more to see specifically what is different in the package.  I just assumed it was platform specific things that would be impossible to remedy but perhaps I was too quick to judge.

Reply:
Was able to fix this with a tweak to the ZipInfo.  Should be able to publish these packages/updates reliably xplat now.